pyhive/tests/test_hive.py

```diff
@@ -72,6 +72,21 @@ def test_complex(self, cursor):
             '0.1',
         ]])
 
+    @with_cursor
+    def test_async(self, cursor):
+        cursor.execute('SELECT * FROM one_row', async=True)
+        while cursor.poll() != ttypes.TOperationState.FINISHED_STATE:
+            cursor.fetch_logs()
+        
+        self.assertEqual(len(cursor.fetchall()), 1)
+    
+    @with_cursor
+    def test_cancel(self, cursor):
+        cursor.execute('SELECT * FROM one_row', async=True)
+        cursor.cancel()
```

Comment:
Oh, I hadn't understood that trick using reflection :P

Nevertheless, it does not work :(

I'll try with MAX()